src/components/Table/Table.js

```diff
@@ -59,8 +62,19 @@ class Table extends Component {
   }
 
   render() {
+    let classes = classNames(
+      'sd-table',
+      'ui',
+      {'very basic': this.props.basic},
+      'selectable',
+      'fixed',
+      'compact',
+      'small',
```

Comment:
Note, the small class should be part of the `size` prop.  This is in our guide line doc, almost finished.  I'll push it up soon.